### PR TITLE
[Issue 2934] Create opp tbls in analytics

### PR DESCRIPTION
**Pull request by babebe**
## Summary
Fixes #{[2934](https://github.com/HHS/simpler-grants-gov/issues/2934)}

## Changes proposed
Migration file to copy initial opportunity tables.

1. lk_opportunity_status
2. lk_opportunity_category
3. opportunity
4. opportunity_summary
5. current_opportunity_summary 

Added documentation for future reference

### CODE REVIEW

**Review by coilysiren — COMMENTED** [review]
LGTM, but what to defer to @chouinar for actual review

File: analytics/src/analytics/integrations/etldb/migrations/versions/0006_add_opportunity_tables.sql [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_id_idx
+    ON  current_opportunity_summary (opportunity_id);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_status_id_idx
+    ON  current_opportunity_summary (opportunity_status_id);
+
+CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_summary_id_idx
+    ON  current_opportunity_summary (opportunity_summary_id);
```
Comment (chouinar) [inline analytics/src/analytics/integrations/etldb/migrations/versions/0006_add_opportunity_tables.sql:129]: Looks like this one was copied twice

```suggestion
CREATE INDEX IF NOT EXISTS current_opportunity_summary_opportunity_status_id_idx
    ON  current_opportunity_summary (opportunity_status_id);
```

Comment (babebe) [reply analytics/src/analytics/integrations/etldb/migrations/versions/0006_add_opportunity_tables.sql:129]: they look similar but one is for `...status_id_idx` and the other is for `...summary_id_idx`

Comment (chouinar) [reply analytics/src/analytics/integrations/etldb/migrations/versions/0006_add_opportunity_tables.sql:129]: Bah, I reread that 3 times and still missed it.

**Review by chouinar — APPROVED** [review]
LGTM!

**Pull request merged**